src/components/app.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import SearchForm from './searchForm';
+import * as locationsArr from '../../locations/ukraine.json';
+
+class App extends React.Component {
+    constructor(props) {
+        super(props);
+
+    }
+
+    render() {
+        return (
+            <div className='app-container'>
+                <SearchForm locations={locationsArr} />
```

Comment:
Passing locations as a prop from here doesn't make sense. Just import them in form component